webpack/assets/javascripts/react_app/redux/middlewares/IntervalMiddleware/__tests__/IntervalMiddleware.test.js

```diff
@@ -57,20 +57,7 @@ describe('Interval Middleware', () => {
 
     IntervalMiddleware(fakeStore)(fakeNext)(stopAction);
     expect(clearInterval).toHaveBeenCalled();
-    expect(fakeNext.mock.calls).toHaveLength(0);
-  });
-
-  it('should handle STOP_INTERVAL action when key does not exist', () => {
-    const fakeStore = getFakeStore();
-    const fakeNext = jest.fn();
-    const stopAction = stopInterval(key);
-
-    try {
-      IntervalMiddleware(fakeStore)(fakeNext)(stopAction);
-    } catch (error) {
-      expect(error.message).toBe(unregisteredIntervalException(key).message);
```

Comment:
test wasn't needed after last refactor where we removed this exception.

Reply:
Look like this test is indeed what caused the timeouts on travis - because it caught the issue of the infinite loop correctly.